bench/commands/make.py

```diff
@@ -19,13 +19,12 @@ def init(path, apps_path, frappe_path, frappe_branch, no_procfile, no_backups,
 
 
 @click.command('get-app')
-@click.argument('name')
 @click.argument('git-url')
 @click.option('--branch', default=None, help="branch to checkout")
-def get_app(name, git_url, branch):
+def get_app(git_url, branch):
```

Comment:
This will break things. So accept the `name` variable for backward compatibility as we discussed. Just add a comment about it.

Or, we will have to fix documentation, setup script, our deployment system, etc.